docs/install-concourse.md

```diff
@@ -4,67 +4,13 @@
 
 ```bash
 cd /data/scripts
-git clone https://github.com/concourse/concourse-docker.git
+git clone https://github.com/trecnoc/concourse-docker.git
+git checkout custom-setup
```

Comment:
Why don't you put it in master like our minio setup?

Reply:
Trying to keep master in line with the upstream

Reply:
That's not consistent/makes our changes harder to find. Since the branch is being deployed I think it should be in master :shrug:

Reply:
OK, will merge the branch back on our master and update the doc